ocamldoc/odoc_info.mli

```diff
@@ -135,7 +135,7 @@ module Name :
 (** Representation and manipulation of method / function / class / module parameters.*)
 module Parameter :
   sig
-    (** {3 Types} *)
+    (** {2 Types} *)
```

Comment:
Why aren't you using `1` here ? The idea is that ocamldoc headers section ocaml modules, so you should restart at `1` here, regardless of whether this is a submodule or not.